app/controllers/prison/dashboards_controller.rb

```diff
@@ -35,23 +35,36 @@ def search
     cancellations
     processed_visits
 
-    @estate = user_estate
+    @estate = current_estate
+  end
+
+  def switch_estate
+    estate = Estate.find(params[:estate_id])
```

Comment:
I decided not to worry about this.

However switching to a now inaccessible estate is handled more gracefully (only possible via an out of date browser tab) – see below.